src/verify.c

```diff
@@ -250,19 +250,16 @@ static int get_required_files(struct manifest *official_manifest, struct list *s
 	int ret;
 
 	if (cmdline_option_install) {
-		progress_set_next_step("download_packs");
 		print("\n");
 		get_all_files(official_manifest, subs);
 	}
 
-	progress_set_next_step("check_files_hash");
+	progress_next_step("check_files_hash", PROGRESS_BAR);
```

Comment:
Just a thought:

Isnt this approach going to add more variability as to the dev deciding whether its going to  PROGRESS_BAR or PROGRESS_UNDEFINED. 

Maybe its too late and a little expensive to think of this, I was thinking maybe we should
use the timer library of c which calls our callback at regular intervals of time.

What I am hinting at is starting a timer at start of an operation and let the kernel calls our callback @ regular intervals for printing progress and then close the progress.

Reply:
> Isnt this approach going to add more variability as to the dev deciding whether its going to PROGRESS_BAR or PROGRESS_UNDEFINED.

I didn't understand your comment. Can you rephrase?

> Maybe its too late and a little expensive to think of this, I was thinking maybe we should
use the timer library of c which calls our callback at regular intervals of time.

It's not too late and it didn't block this PR. I'm ok on improving the spinner to be displayed even when there's not internet download. It's better than what we have now. But it's low priority though.

Reply:
What I meant there can be a confusion about an operation whether it is need to report PROGRESS_BAR or PROGRESS_UNDEFINED.  But don't feel strongly about this as at-least now there is a way to classify an operation.